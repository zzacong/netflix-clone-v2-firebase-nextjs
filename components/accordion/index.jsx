import { useState, useContext, createContext } from 'react'
import { MdAdd, MdClose } from 'react-icons/md'
import {
  Container,
  Frame,
  Title,
  Item,
  Inner,
  Header,
  Body,
} from './styles/accordion'

const ToggleContext = createContext()

export default function Accordion({ children, ...props }) {
  return (
    <Container {...props}>
      <Inner>{children}</Inner>
    </Container>
  )
}

Accordion.Title = ({ children, ...props }) => {
  return <Title {...props}>{children}</Title>
}

Accordion.Frame = ({ children, ...props }) => {
  return <Frame {...props}>{children}</Frame>
}

Accordion.Item = ({ children, ...props }) => {
  const [toggleShow, setToggleShow] = useState(false)

  return (
    <ToggleContext.Provider value={{ toggleShow, setToggleShow }}>
      <Item {...props}>{children}</Item>
    </ToggleContext.Provider>
  )
}

Accordion.Header = ({ children, ...props }) => {
  const { toggleShow, setToggleShow } = useContext(ToggleContext)

  return (
    <Header onClick={() => setToggleShow(prev => !prev)} {...props}>
      {children}
      {toggleShow ? <MdClose /> : <MdAdd />}
    </Header>
  )
}

Accordion.Body = ({ children, ...props }) => {
  const { toggleShow } = useContext(ToggleContext)

  return (
    <Body className={toggleShow ? 'open' : 'closed'} {...props}>
      <span>{children}</span>
    </Body>
  )
}
