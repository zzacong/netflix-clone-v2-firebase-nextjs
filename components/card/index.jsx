import React, { useState, useContext, createContext } from 'react'
import capitalize from 'lodash.capitalize'
import { MdClose } from 'react-icons/md'
import {
  Container,
  Group,
  Title,
  Subtitle,
  Text,
  Feature,
  FeatureTitle,
  FeatureText,
  FeatureClose,
  Maturity,
  Content,
  Meta,
  Entities,
  Item,
  Image,
} from './styles/card'

export const FeatureContext = createContext()

export default function Card({ children, ...props }) {
  const [showFeature, setShowFeature] = useState(false)
  const [itemFeature, setItemFeature] = useState({})

  return (
    <FeatureContext.Provider
      value={{ showFeature, setShowFeature, itemFeature, setItemFeature }}
    >
      <Container {...props}>{children}</Container>
    </FeatureContext.Provider>
  )
}

Card.Group = ({ children, ...props }) => <Group {...props}>{children}</Group>

Card.Title = ({ children, ...props }) => <Title {...props}>{children}</Title>

Card.Subtitle = ({ children, ...props }) => (
  <Subtitle {...props}>{children}</Subtitle>
)

Card.Text = ({ children, ...props }) => <Text {...props}>{children}</Text>

Card.Entities = ({ children, ...props }) => (
  <Entities {...props}>{children}</Entities>
)

Card.Meta = ({ children, ...props }) => <Meta {...props}>{children}</Meta>

Card.Item = ({ item, children, ...props }) => {
  const { setShowFeature, setItemFeature } = useContext(FeatureContext)

  return (
    <Item
      onClick={() => {
        setItemFeature(item)
        setShowFeature(true)
      }}
      {...props}
    >
      {children}
    </Item>
  )
}

Card.Image = ({ ...props }) => <Image {...props} />

Card.Feature = function CardFeature({ children, category, ...props }) {
  const { showFeature, itemFeature, setShowFeature } = useContext(
    FeatureContext
  )

  if (!showFeature) return null

  return (
    <Feature
      {...props}
      src={`/images/${category}/${itemFeature.genre}/${itemFeature.slug}/large.jpg`}
    >
      <Content>
        <FeatureTitle>{itemFeature.title}</FeatureTitle>
        <FeatureText>{itemFeature.description}</FeatureText>
        <FeatureClose onClick={() => setShowFeature(false)}>
          <MdClose />
        </FeatureClose>

        <Group margin="30px 0" flexDirection="row" alignItems="center">
          <Maturity rating={itemFeature.maturity}>
            {itemFeature.maturity < 12 ? 'PG' : itemFeature.maturity}
          </Maturity>
          <FeatureText fontWeight="bold">
            {capitalize(itemFeature.genre)}
          </FeatureText>
        </Group>

        {children}
      </Content>
    </Feature>
  )
}
