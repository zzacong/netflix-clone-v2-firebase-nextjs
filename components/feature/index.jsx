import { Container, Title, SubTitle } from './styles/feature'

export default function Feature({ children, ...props }) {
  return <Container {...props}>{children}</Container>
}

Feature.Title = ({ children, ...props }) => <Title {...props}>{children}</Title>

Feature.SubTitle = ({ children, ...props }) => (
  <SubTitle {...props}>{children}</SubTitle>
)
