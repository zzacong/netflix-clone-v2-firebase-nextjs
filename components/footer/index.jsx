import {
  Container,
  Row,
  Column,
  Link,
  Title,
  Text,
  Break,
} from './styles/footer'

export default function Footer({ children, ...props }) {
  return <Container {...props}>{children}</Container>
}

Footer.Row = ({ children, ...props }) => {
  return <Row {...props}>{children}</Row>
}

Footer.Column = ({ children, ...props }) => {
  return <Column {...props}>{children}</Column>
}

Footer.Link = ({ children, ...props }) => {
  return <Link {...props}>{children}</Link>
}

Footer.Title = ({ children, ...props }) => {
  return <Title {...props}>{children}</Title>
}

Footer.Text = ({ children, ...props }) => {
  return <Text {...props}>{children}</Text>
}

Footer.Break = ({ ...restProps }) => {
  return <Break {...restProps} />
}
