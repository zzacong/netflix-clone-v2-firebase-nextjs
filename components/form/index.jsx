import { forwardRef } from 'react'
import NextLink from 'next/link'
import {
  Container,
  Error,
  Base,
  Title,
  Text,
  TextSmall,
  Link,
  Input,
  Submit,
} from './styles/form'

export default function Form({ children, ...props }) {
  return <Container {...props}>{children}</Container>
}

Form.Base = ({ children, ...props }) => <Base {...props}>{children}</Base>

Form.Error = ({ children, ...props }) => <Error {...props}>{children}</Error>

Form.Title = ({ children, ...props }) => <Title {...props}>{children}</Title>

Form.Text = ({ children, ...props }) => <Text {...props}>{children}</Text>

Form.TextSmall = ({ children, ...props }) => (
  <TextSmall {...props}>{children}</TextSmall>
)

Form.Link = ({ href, children, ...props }) => (
  <NextLink href={href} passHref>
    <Link {...props}>{children}</Link>
  </NextLink>
)

Form.Input = forwardRef(({ children, ...props }, ref) => (
  <Input {...props} ref={ref}>
    {children}
  </Input>
))

Form.Submit = ({ children, ...props }) => <Submit {...props}>{children}</Submit>
