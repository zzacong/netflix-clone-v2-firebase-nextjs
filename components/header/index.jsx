import { useState } from 'react'
import NextLink from 'next/link'
import { MdInfoOutline, MdSearch } from 'react-icons/md'
import { IoMdPlay } from 'react-icons/io'
import {
  Container,
  Group,
  Background,
  Dropdown,
  Picture,
  Link,
  Search,
  Profile,
  FeatureCallOut,
  SearchIcon,
  SearchInput,
  ButtonLink,
  PlayButton,
  InfoButton,
  Text,
  Feature,
  Logo,
} from './styles/header'

export default function Header({ bg = true, children, ...props }) {
  return bg ? <Background {...props}>{children}</Background> : children
}

Header.Frame = ({ children, ...props }) => (
  <Container {...props}>{children}</Container>
)

Header.Group = ({ children, ...props }) => <Group {...props}>{children}</Group>

Header.Logo = ({ href, ...props }) => (
  <NextLink href={href} passHref>
    <a>
      <Logo {...props} />
    </a>
  </NextLink>
)

Header.ButtonLink = ({ href, children, ...props }) => (
  <NextLink href={href} passHref>
    <ButtonLink {...props}>{children}</ButtonLink>
  </NextLink>
)

Header.Feature = ({ children, ...props }) => (
  <Feature {...props}>{children}</Feature>
)

Header.FeatureCallOut = ({ children, ...props }) => (
  <FeatureCallOut {...props}>{children}</FeatureCallOut>
)

Header.Text = ({ children, ...props }) => <Text {...props}>{children}</Text>

Header.TextLink = ({ children, ...props }) => <Link {...props}>{children}</Link>

Header.Profile = ({ children, ...props }) => (
  <Profile {...props}>{children}</Profile>
)

Header.Picture = ({ src, ...props }) => (
  <Picture src={`/images/users/${src}.png`} {...props} />
)

Header.Dropdown = ({ children, ...props }) => (
  <Dropdown {...props}>{children}</Dropdown>
)

Header.Search = ({ searchTerm, setSearchTerm, ...props }) => {
  const [searchActive, setSearchActive] = useState(false)

  return (
    <Search active={searchActive} {...props}>
      <SearchIcon onClick={() => setSearchActive(prev => !prev)}>
        <MdSearch />
      </SearchIcon>
      <SearchInput
        value={searchTerm}
        onChange={e => setSearchTerm(e.target.value)}
        placeholder="Films, series"
        active={searchActive}
      />
    </Search>
  )
}

Header.PlayButton = ({ children, ...props }) => (
  <PlayButton {...props}>
    <IoMdPlay />
    {children}
  </PlayButton>
)

Header.InfoButton = ({ children, ...props }) => (
  <InfoButton {...props}>
    <MdInfoOutline />
    {children}
  </InfoButton>
)
