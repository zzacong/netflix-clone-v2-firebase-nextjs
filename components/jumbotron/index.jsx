import {
  Inner,
  Container,
  Pane,
  Title,
  SubTitle,
  Image,
  Item,
} from './styles/jumbotron'

export default function Jumbotron({ children, direction = 'row', ...props }) {
  return (
    <Item direction={direction}>
      <Inner>{children}</Inner>
    </Item>
  )
}

Jumbotron.Container = ({ children, ...props }) => {
  return <Container {...props}>{children}</Container>
}
Jumbotron.Pane = ({ children, ...props }) => {
  return <Pane {...props}>{children}</Pane>
}
Jumbotron.Title = ({ children, ...props }) => {
  return <Title {...props}>{children}</Title>
}
Jumbotron.SubTitle = ({ children, ...props }) => {
  return <SubTitle {...props}>{children}</SubTitle>
}
Jumbotron.Image = ({ children, ...props }) => {
  return <Image {...props} />
}
