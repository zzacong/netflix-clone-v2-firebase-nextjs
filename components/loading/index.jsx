import { LockBody, ReleaseBody, Spinner, Picture } from './styles/loading'

export default function Loading({ src, ...props }) {
  return (
    <Spinner {...props}>
      <LockBody />
      <Picture src={`/images/users/${src}.png`} />
    </Spinner>
  )
}

Loading.ReleaseBody = () => <ReleaseBody />
