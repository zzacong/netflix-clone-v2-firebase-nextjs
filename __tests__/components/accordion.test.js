import { render, fireEvent, getDefaultNormalizer } from '@testing-library/react'
import { Accordion } from '../../components'
import faqData from '../../fixtures/faqs.json'

describe('<Accordion />', () => {
  test('renders the <Accordion /> with populated data', () => {
    const { container, getByText } = render(
      <Accordion>
        <Accordion.Title>Frequently Asked Question</Accordion.Title>
        <Accordion.Frame>
          {faqData.map(item => (
            <Accordion.Item key={item.id}>
              <Accordion.Header>{item.header}</Accordion.Header>
              <Accordion.Body>{item.body}</Accordion.Body>
            </Accordion.Item>
          ))}
        </Accordion.Frame>
      </Accordion>
    )

    expect(getByText('Frequently Asked Question')).toBeInTheDocument()
    expect(getByText('What is Netflix?')).toBeInTheDocument()
    expect(getByText('How much does Netflix cost?')).toBeInTheDocument()
    expect(getByText('Where can I watch?')).toBeInTheDocument()
    expect(getByText('How do I cancel?')).toBeInTheDocument()
    expect(getByText('What can I watch on Netflix?')).toBeInTheDocument()
    expect(container.firstChild).toMatchSnapshot()
  })
})

test('opens and closes the <Accordion /> component', () => {
  const { container, queryByText } = render(
    <Accordion>
      <Accordion.Title>Frequently Asked Question</Accordion.Title>
      <Accordion.Frame>
        {faqData.map(item => (
          <Accordion.Item key={item.id}>
            <Accordion.Header>{item.header}</Accordion.Header>
            <Accordion.Body>{item.body}</Accordion.Body>
          </Accordion.Item>
        ))}
      </Accordion.Frame>
    </Accordion>
  )

  const whatIsNetflixBodyText = faqData.find(({ id }) => id === 1).body
  const fn = () =>
    queryByText(whatIsNetflixBodyText, {
      normalizer: getDefaultNormalizer({ collapseWhitespace: false }),
    })

  expect(fn()).not.toBeVisible()
  fireEvent.click(queryByText('What is Netflix?'))
  expect(fn()).toBeVisible()
  fireEvent.click(queryByText('What is Netflix?'))
  expect(fn()).not.toBeVisible()

  expect(container.firstChild).toMatchSnapshot()
})
