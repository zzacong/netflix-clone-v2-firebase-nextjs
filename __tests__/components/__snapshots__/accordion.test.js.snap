// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Accordion /> renders the <Accordion /> with populated data 1`] = `
<div
  class="accordion__Container-sc-1drqh43-0 iJkhvl"
>
  <div
    class="accordion__Inner-sc-1drqh43-2 gLhaei"
  >
    <h2
      class="accordion__Title-sc-1drqh43-3 gKkuyd"
    >
      Frequently Asked Question
    </h2>
    <div
      class="accordion__Frame-sc-1drqh43-1 bwKqDM"
    >
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          What is Netflix?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Netflix is a streaming service that offers a wide variety of award-winning TV programmes, films, anime, documentaries and more – on thousands of internet-connected devices.

You can watch as much as you want, whenever you want, without a single advert – all for one low monthly price. There's always something new to discover, and new TV programmes and films are added every week!
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          How much does Netflix cost?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Watch Netflix on your smartphone, tablet, smart TV, laptop or streaming device, all for one low fixed monthly fee. Plans start from £5.99 a month. No extra costs or contracts.
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          Where can I watch?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Watch anywhere, anytime, on an unlimited number of devices. Sign in with your Netflix account to watch instantly on the web at netflix.com from your personal computer or on any internet-connected device that offers the Netflix app, including smart TVs, smartphones, tablets, streaming media players and game consoles.

You can also download your favourite programmes with the iOS, Android, or Windows 10 app. Use downloads to watch while you're on the go and without an internet connection. Take Netflix with you anywhere.
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          How do I cancel?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Netflix is flexible. There are no annoying contracts and no commitments. You can easily cancel your account online in two clicks. There are no cancellation fees – start or stop your account at any time.
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          What can I watch on Netflix?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Netflix has an extensive library of feature films, documentaries, TV programmes, anime, award-winning Netflix originals, and more. Watch as much as you want, any time you want.
          </span>
        </article>
      </section>
    </div>
  </div>
</div>
`;

exports[`opens and closes the <Accordion /> component 1`] = `
<div
  class="accordion__Container-sc-1drqh43-0 iJkhvl"
>
  <div
    class="accordion__Inner-sc-1drqh43-2 gLhaei"
  >
    <h2
      class="accordion__Title-sc-1drqh43-3 gKkuyd"
    >
      Frequently Asked Question
    </h2>
    <div
      class="accordion__Frame-sc-1drqh43-1 bwKqDM"
    >
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          What is Netflix?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Netflix is a streaming service that offers a wide variety of award-winning TV programmes, films, anime, documentaries and more – on thousands of internet-connected devices.

You can watch as much as you want, whenever you want, without a single advert – all for one low monthly price. There's always something new to discover, and new TV programmes and films are added every week!
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          How much does Netflix cost?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Watch Netflix on your smartphone, tablet, smart TV, laptop or streaming device, all for one low fixed monthly fee. Plans start from £5.99 a month. No extra costs or contracts.
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          Where can I watch?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Watch anywhere, anytime, on an unlimited number of devices. Sign in with your Netflix account to watch instantly on the web at netflix.com from your personal computer or on any internet-connected device that offers the Netflix app, including smart TVs, smartphones, tablets, streaming media players and game consoles.

You can also download your favourite programmes with the iOS, Android, or Windows 10 app. Use downloads to watch while you're on the go and without an internet connection. Take Netflix with you anywhere.
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          How do I cancel?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Netflix is flexible. There are no annoying contracts and no commitments. You can easily cancel your account online in two clicks. There are no cancellation fees – start or stop your account at any time.
          </span>
        </article>
      </section>
      <section
        class="accordion__Item-sc-1drqh43-4 ksozVe"
      >
        <header
          class="accordion__Header-sc-1drqh43-5 rDirl"
        >
          What can I watch on Netflix?
          <svg
            fill="currentColor"
            height="1em"
            stroke="currentColor"
            stroke-width="0"
            viewBox="0 0 24 24"
            width="1em"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M19 13h-6v6h-2v-6H5v-2h6V5h2v6h6v2z"
            />
          </svg>
        </header>
        <article
          class="accordion__Body-sc-1drqh43-6 cqceYJ closed"
        >
          <span>
            Netflix has an extensive library of feature films, documentaries, TV programmes, anime, award-winning Netflix originals, and more. Watch as much as you want, any time you want.
          </span>
        </article>
      </section>
    </div>
  </div>
</div>
`;
